src/coreclr/jit/CMakeLists.txt

```diff
@@ -465,6 +456,10 @@ function(add_jit jitName)
         ${JIT_ARCH_LINK_LIBRARIES}
     )
 
+    if (CLR_CMAKE_HOST_WIN32)
+        link_natvis_sources_for_target(${jitName} PRIVATE clrjit.natvis)
```

Comment:
what is the purpose of `PRIVATE`?

Reply:
The `PRIVATE` makes sure to keep the natvis linked only to the jit dlls and not to dlls that link to the jit dlls in the CMake build. See here in the CMake docs: https://cmake.org/cmake/help/latest/command/target_link_libraries.html#id3